### PR TITLE
Drop CLI from the crate

### LINKED ISSUE #126: Dropping the CLI
I was thinking about removing the CLI entirely. It's a bit of maintenance and I doubt very much anybody uses it at all. I did it as a pet project back in the days to learn Rust. Nowadays I believe the lib is still actually used but not the CLI itself.

Since all the dependencies of the CLI are behind a feature flag, I think it can be easily removed without breaking compatiblity for anyone.

What do you think @bgilbert?

### PR DESCRIPTION
**Pull request by cecton**
Fixes #126

### CODE REVIEW

**Comment by cecton** [comment]
I'm not 100% sure about the category. The closest one is also definitely not what this library is:

> filesystem - Crates for dealing with files and filesystems.

The complete list is there: https://crates.io/category_slugs

File: README.md [hunk 1]
```diff
@@ -9,68 +9,41 @@
...
+ *  Fix partitions order
+ *  Change disk GUID
+ *  Change partition name
+ *  Change partition GUID
+ *  Toggle legacy BIOS bootable
+ *  Toggle no block IO protocol
+ *  Toggle required partition flag
+ *  Toggle attributes
+ *  Swap partition indexes
+ *  Copy/clone all partitions from one disk and insert it to another
```
Comment (bgilbert) [inline README.md:33]: A lot of these aren't really features of the library, but are just things you can do by manipulating GPT data structures.  Maybe the list is better framed as "ideas" or "things you can do with gptman"?

File: Cargo.toml [hunk 2]
```diff
@@ -5,14 +5,14 @@ authors = ["Cecile Tonglet <cecile.tonglet@cecton.com>"]
 license = "MIT OR Apache-2.0"
 edition = "2021"
 rust-version = "1.63"
-description = "A GPT manager that allows you to copy partitions from one disk to another"
+description = "Pure-Rust library to work with GUID partition tables."
```
Comment (bgilbert) [inline Cargo.toml:8]: How about:

```suggestion
description = "Pure Rust library to read and modify GUID partition tables"
```

File: README.md [hunk 3]
```diff
@@ -9,68 +9,41 @@
 gptman
 ======
 
-A library that allows managing GUID partition tables.
-
-A CLI tool for Linux that allows you to copy a partition from one disk to
-another and more.
+Pure-Rust library to work with GUID partition tables.
```
Comment (bgilbert) [inline README.md:12]: Same.

**Review by bgilbert — COMMENTED** [review]
LGTM generally!  This can also drop:

- the `update.yml` workflow, since there's no longer a `Cargo.lock` to update
- the Rust 1.70.0 runs from `rust.yml`, since we won't have a separate CLI MSRV
- the `--all-features` flag in CI, since there won't be any features

> I'm not 100% sure about the category. The closest one is also definitely not what this library is:

Yeah, that list has some surprising omissions for a systems language.  I agree that `filesystem` is the best option available.

File: README.md (hunk 1 above)
Comment (cecton) [reply README.md:33]: Yes they were actually the feature of the CLI. I will rename the title

**Comment by cecton** [comment]
> the `--all-features` flag in CI, since there won't be any features

There is still currently a feature for "nix" dependency

File: README.md (hunk 3 above)
Comment (cecton) [reply README.md]: Fixed in f0f3955ca6f5879719f0236362b26fdf67d492bf

**Pull request merged**